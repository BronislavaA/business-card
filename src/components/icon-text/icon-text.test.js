import React from 'react';
import renderer from 'react-test-renderer';
import IconText from './icon-text'

test('renders IconText component', () => {
  const component = renderer.create(
    <IconText phone/>,
    <IconText address/>,
    <IconText email/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});