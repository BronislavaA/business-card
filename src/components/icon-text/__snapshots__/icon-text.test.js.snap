// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders IconText component 1`] = `
<div
  className="sc-bdVaJa kaWZKk"
>
  <div>
    <i
      class="material-icons"
    >
      phone
    </i>
    <p>
      <b>
        +477 3086 7958
      </b>
    </p>
  </div>
</div>
`;
